channel_test.go

```diff
@@ -95,6 +95,25 @@ func TestChannel_AddMembers(t *testing.T) {
 	assert.Equal(t, user.ID, ch.Members[0].User.ID, "members contain user id")
 }
 
+func TestChannel_AssignRoles(t *testing.T) {
+	c := initClient(t)
+
+	owner := randomUser(t, c)
+	chanID := randomString(12)
+
+	ch, err := c.CreateChannel("messaging", chanID, owner.ID, map[string]interface{}{
+		"members": []string{owner.ID},
+	})
+	require.NoError(t, err, "create channel")
+	defer func() {
+		_ = ch.Delete()
+	}()
```

Comment:
You can use [t.Cleanup](https://pkg.go.dev/testing#T.Cleanup) for this kind of thing.

Reply:
Yes, but I wanted to have it consistent with the rest of the tests here